task_templates/transforms/python3_image_transform/custom.py

```diff
@@ -0,0 +1,69 @@
+import pickle
+import pandas as pd
+from pathlib import Path
+
+from img_utils import (
+    b64_to_img,
+    img_to_b64,
+    img_to_grayscale,
+)
+
+
+def _process_image(raw_data):
```

Comment:
can you add type annotations here, and in img_utils? Might help with readability.